src/vivarium_gates_iv_iron/components/pregnancy.py

```diff
@@ -171,16 +251,21 @@ def on_simulation_end(self):
     def load_pregnancy_prevalence(self, builder: Builder) -> pd.DataFrame:
         index_cols = [col for col in metadata.ARTIFACT_INDEX_COLUMNS if col != 'location']
         not_pregnant_prevalence = (builder.data.load(data_keys.PREGNANCY.NOT_PREGNANT_PREVALENCE)
-                               .fillna(0)
-                               .set_index(index_cols))
+                                   .fillna(0)
+                                   .set_index(index_cols))
         pregnant_prevalence = (builder.data.load(data_keys.PREGNANCY.PREGNANT_PREVALENCE)
                                .fillna(0)
                                .set_index(index_cols))
         postpartum_prevalence = (builder.data.load(data_keys.PREGNANCY.POSTPARTUM_PREVALENCE)
-                               .fillna(0)
-                               .set_index(index_cols))
-        # order of prevalences must match order of PREGNANCY_STATUSES
-        prevalences = pd.concat([not_pregnant_prevalence, pregnant_prevalence, postpartum_prevalence], axis=1)
+                                 .fillna(0)
+                                 .set_index(index_cols))
+        maternal_disorder_prevalence = pd.Series(0., index=postpartum_prevalence.index, name=models.MATERNAL_DISORDER_STATE)
+        no_maternal_disorder_prevalence = 1/6 * postpartum_prevalence
```

Comment:
These `1/6` and `5/6` values should probably be defined in the `data_values` file.

Reply:
Agree